### PR TITLE
chore: update docs links

### PR DESCRIPTION
**Pull request by itsacoyote**
Update broken links to working URLs. 
Update links referencing the ZKsync docs and correctly reference ZKsync lite docs.

### CODE REVIEW

File: docs/dev/contracts/README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ based on a native Ethereum signature, all wallets, even hardware wallets, will w
 
 ### Deployment
 
-zkSync Era Mainnet is here! To get started visit the [zkSync Era Documentation](https://era.zksync.io/docs/).
+zkSync Era Mainnet is here! To get started visit the [ZKsync Era Documentation](https://docs.zksync.io).
```
Comment (jennyg0) [inline docs/dev/contracts/README.md]: zkSync here

Comment (itsacoyote) [reply docs/dev/contracts/README.md]: Good catch 👍🏻

File: docs/dev/README.md [hunk 2]
```diff
@@ -36,4 +36,4 @@ Thirdly, zkSync has native support of NFTs. You can try it out in our [wallet](h
 Finally, zkSync support is implemented for all the main platforms. Check out our [SDK section](/api/sdk) of docs, and
 start developing with zkSync!
 
-If you need smart contract support, zkSync Era is your go to solution. As of February 2022, zkSync Era Testnet has been live featuring smart contracts in Solidity or reusing existing Solidity code. You can find more details in the [zkSync Era Documentation](https://era.zksync.io/docs)
+If you need smart contract support, zkSync Era is your go to solution. As of February 2022, zkSync Era Testnet has been live featuring smart contracts in Solidity or reusing existing Solidity code. You can find more details in the [zkSync Era Documentation](https://docs.zksync.io)
```
Comment (jennyg0) [inline docs/dev/README.md:39]: zkSync here x3

Comment (itsacoyote) [reply docs/dev/README.md:39]: Technically it's `zkSync` all across the docs, once I edit one I'll end up editing everything.
I'm not sure if we should just go through the entire project and update it, I'll consult with the team.

**Review by jennyg0 — APPROVED** [review]

File: docs/dev/README.md (hunk 2 above)
Comment (jennyg0) [reply docs/dev/README.md:39]: true, will approve for the links

**Pull request merged**